src/agent/onefuzz-agent/src/tasks/heartbeat.rs

```diff
@@ -32,12 +32,13 @@ pub struct TaskContext {
 
 pub type TaskHeartbeatClient = HeartbeatClient<TaskContext, HeartbeatData>;
 
-pub async fn init_task_heartbeat(queue_url: Url, task_id: Uuid) -> Result<TaskHeartbeatClient> {
+pub async fn init_task_heartbeat(queue_url: Url, task_id: Uuid, job_id: Uuid) -> Result<TaskHeartbeatClient> {
```

Comment:
Please run `cargo fmt` on the code.  (This suggestion implements the required change)

```suggestion
pub async fn init_task_heartbeat(
    queue_url: Url,
    task_id: Uuid,
    job_id: Uuid,
) -> Result<TaskHeartbeatClient> {
```